### DISCUSSION TITLE
Queues for on maintenance sites

### DISCUSSION THREAD

**Discussion by ezerw** [laravel/framework#44191] [General]
# Hello 👋 

Just wanted to say that it could be nice if the Queues documentation mentioned somewhere that if the site is under maintenance (`php artisan down`) the jobs are not processed.

Basically, I have a site that runs 2 environments in the same VPS and I spent around 3 hours trying to figure out why one environment (test) was processing the jobs fine and the other not considering they both share almost the same configuration in supervisor.

After trying:
- Use different Redis DB for each environment
- Make sure `APP_NAME` is unique for each
- Use `Database` queue connection for test and `Redis` for prod
- Tweaked tons of times Supervisor config

And a few others I don't remember I ended up reading the `php artisan queue:work` help which mentions that you need to add `--force` to run even in maintenance mode.

Sorry and please ignore if it's mentioned already or if there is a doc I missed 🙏

**Comment by vaites** [comment]
For those of you who get here, this has already been documented: https://laravel.com/docs/12.x/queues#maintenance-mode-queues
